services/svc/cosmosdb/autorust.toml

```diff
@@ -0,0 +1,4 @@
+[tags]
+deny = [
+  "package-2019-02"
```

Comment:
Per discussion on teams, the current spec for this tag causes generation to fail with the error:

```
8 cosmos-db  package-2019-02Error: Error { context: Custom(Custom { kind: DataConversion, error: Error("data did not match any variant of untagged enum ReferenceOr", line: 1209, column: 5) }) }
```